database/schema.sql

```diff
@@ -13,4 +13,23 @@ create table "public"."users" (
   "createdAt"      timestamptz(6) not null default now(),
   primary key ("userId"),
   unique ("username")
+
+);
+
+create table "public"."bookmarks" (
+  "userId"         integer         not null,
+  "businessId"     text           not null,
+  "image"          text           not null,
+  "name"           text           not null,
+  "rating"         text           not null,
+  "address1"       text           not null,
+  "address2"       text           not null,
+  "city"           text           not null,
+  "state"          text           not null,
+  "zipcode"        text           not null,
+  "latitude"       text           not null,
+  "longitude"      text           not null,
+  "createdAt"      timestamptz(6) not null default now(),
+  primary key ("businessId"),
+  unique ("businessId")
```

Comment:
This `unique` constraint is redundant. `primary key` implicitly imposes a `unique` constraint.